### DISCUSSION TITLE
MIDI Keyboard

### DISCUSSION THREAD

**Discussion by ghost** [YuriSizov/boscaceoil-blue#31] [Ideas]
From what I see here and in the previous Bosca Ceoil version, there was no MIDI keyboard support. It would be amazing if that were added. For  bosca ceoil version there was no MIDI keyboard support it would be amazing if that is added

**Comment by YuriSizov** [comment]
Yes, I'd love to have some MIDI input devices support in Bosca. I don't own any right now, so it would be hard for me to implement and test it properly, but it's definitely something on the TODO list!
